presets/4.3/tune/mouseFPV_FFVCycle_Incisor_PT5_tune_filters.txt

```diff
@@ -0,0 +1,242 @@
+#$ TITLE:  FPVCycle Incisor/Prototype 5 6s Tune | mouseFPV 
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: TUNE
+#$ STATUS: EXPERIMENTAL
+#$ KEYWORDS: FPVCycle
```

Comment:
Configurator user keywords for string search when user types something in the search bar (besides title and author name)
So adding more keywords would allow users to find your preset by "5inch" or something like that:
5", 5 inch, 5in, 6S, freestyle, GoPro